assignments/006-simple-snakemake-maneuvers/Snakefile

```diff
@@ -1,19 +1,19 @@
 
 
 # run it over 3 samples
-SAMPLES = ['A', 'B', 'C']
+SAMPLES = ['DPCh_plate1_B10_S22', 'DPCh_plate1_B11_S23', 'DPCh_plate1_B12_S24', 'DPCh_plate1_C10_S34', 'DPCh_plate1_C11_S35', 'DPCh_plate1_C12_S36', 'DPCh_plate1_D09_S45', 'DPCh_plate1_D11_S47', 'DPCh_plate1_F10_S70', 'DPCh_plate1_F11_S71', 'DPCh_plate1_F12_S72', 'DPCh_plate1_G09_S81', 'DPCh_plate1_G10_S82', 'DPCh_plate1_G12_S84', 'DPCh_plate1_H09_S93', 'DPCh_plate1_H10_S94']
 
 # variant calling over two "chromosomes"
-CHROMOS = [ "NC_037122.1f5t9", "NC_037123.1f10t14"]
+CHROMOS = ["NC_037122.1f5t9", "NC_037123.1f10t14", "NC_037124.1f8t16", "NC_037125.1f20t24"]
 
 
 rule genome_faidx:
   input:
-    "resources/genome.fasta",
+    "data/genome/genome.fasta",
   output:
-    "resources/genome.fasta.fai",
+    "data/genome/genome.fasta.fai",
   conda:
-    "envs/bwa2sam.yaml"
+    "assignments/006-simple-snakemake-maneuvers/envs/bwa2sam.yaml"
```

Comment:
Hey! I am not sure that this would work if you ran it not as a dry run.  My understanding is that the path to the envs must be given relative to the position of the Snakefile, not the current working directory.  Did you try running it?